lib/google_analytics/google_analytics_export_credentials.rb

```diff
@@ -0,0 +1,15 @@
+require 'googleauth'
+
+module GoogleAnalytics
+  class GoogleAnalyticsExportCredentials
+    def self.authorization(scopes)
+      raise ArgumentError, "Must define GOOGLE_PRIVATE_KEY and GOOGLE_CLIENT_EMAIL in order to authenticate." unless all_configuration_in_env?
+      Google::Auth.get_application_default(scopes)
+    end
+
+    def self.all_configuration_in_env?
+      %w(GOOGLE_PRIVATE_KEY GOOGLE_CLIENT_EMAIL).all? { |env_var| ENV[env_var].present? }
+    end
```

Comment:
Maybe we should add `GOOGLE_ACCOUNT_TYPE` to these lists?

Reply:
I've moved the setting of this var inside `GoogleAnalyticsExportCredentials`, so now we always have it set.

Reply:
:+1: